build.gradle

```diff
@@ -78,18 +79,19 @@ githubRelease {
     owner = 'gradle'
     repo = 'common-custom-user-data-gradle-plugin'
     targetCommitish = 'main'
-    releaseName = gitHubReleaseName()
-    tagName = gitReleaseTag()
+    releaseName = releaseVersion
+    tagName = releaseVersion.map { "v$it" }
     prerelease = false
     overwrite = false
     generateReleaseNotes = false
     body = releaseNotes
+    dryRun = true
```

Comment:
Why was this added?

Reply:
Was planning on removing this before merging, I've gone ahead and removed it.